### PR TITLE
Add missing vacuum toggle service

### PR DESCRIPTION
**Pull request by jpbede**
## Proposed change
https://github.com/home-assistant/core/pull/106729 added the missing vacuum toggle service to the `services.yaml`, the vacuum doc also misses the toggle service, so add this as well.

As the PR above will be in the 2024.1 release this should be cherry-picked in to the `rc`.

## Type of change

- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [ ] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [ ] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [ ] I've opened up a PR to add logo's and icons in [Brands repository](https://github.com/home-assistant/brands).
- [x] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase: https://github.com/home-assistant/core/pull/106729
- Link to parent pull request in the Brands repository: 
- This PR fixes or closes issue: fixes #

## Checklist

- [x] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [x] The documentation follows the Home Assistant documentation [standards].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

### CODE REVIEW

File: source/_integrations/vacuum.markdown [hunk 1]
```diff
@@ -45,6 +45,14 @@ Stop the current activity of the vacuum.
...
 | `entity_id`            | yes      | Only act on specific vacuum. Use `entity_id: all` to target all. |
 
+### Service `vacuum.toggle`
+
+Toggles the vacuum cleaner on/off.
+
+| Service data attribute | Optional | Description                                                      |
+| ---------------------- | -------- | ---------------------------------------------------------------- |
+| `entity_id`            | yes      | Only act on specific vacuum. Use `entity_id: all` to target all. |
+
```
Comment (davet2001) [inline source/_integrations/vacuum.markdown:55]: If this attribute is optional, what is the behaviour if it is omitted? Does that need to be clarified in the documentation?

**Review by davet2001 — CHANGES_REQUESTED** [review]

File: source/_integrations/vacuum.markdown (hunk 1 above)
Comment (jpbede) [reply source/_integrations/vacuum.markdown:55]: Well, I haven't check how this is handled, but `entity_id` is also optional for any other entity service, like `humidifier.toggle`

File: source/_integrations/vacuum.markdown [hunk 2]
```diff
@@ -45,6 +45,14 @@ Stop the current activity of the vacuum.
 | ---------------------- | -------- | ---------------------------------------------------------------- |
 | `entity_id`            | yes      | Only act on specific vacuum. Use `entity_id: all` to target all. |
 
+### Service `vacuum.toggle`
```
Comment (MartinHjelmare) [inline source/_integrations/vacuum.markdown:48]: This service is deprecated and support will be completely removed in 2024.2. Not sure we should add it now.

Comment (jpbede) [reply source/_integrations/vacuum.markdown:48]: If we remove it either way, then I wouldn't add that either. I then close the PRa

**Pull request closed without merging**